geemusic/intents/selection.py

```diff
@@ -94,6 +95,51 @@ def play_artist_radio(artist_name):
     speech_text = "Playing %s radio" % artist['name']
     return audio(speech_text).play(stream_url)
 
+@ask.intent("GeeMusicPlayPlaylistIntent")
+def play_playlist(playlist_name):
```

Comment:
I'm not real good with the map and filter commands and could not figure out how to use them in a try, except loop. If you know how please let me know.